src/main/resources/static/style/style.css

```diff
@@ -34,18 +35,20 @@ body {
   z-index: -1;
 }
 
+
```

Comment:
there should only be one linebreak between css rules. 

Please clean up for the next PR